docs/section22/Rule22-8.md

```diff
@@ -0,0 +1,54 @@
+
+# CHW&CW - Rule 22-8  
+
+**Rule ID:** 22-8  
+**Rule Description:** For Baseline chilled water system with cooling capacity of 300 tons or more, the secondary pump shall be modeled with variable-speed drives.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.10 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13.
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+3. Pass Rule-22-7.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13: `PLACEHOLDER`
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water: `if B_RMR.ASHRAE229.chillers:`
+3. Pass Rule-22-7: `if rule-22-7 == "PASS":`
+
+## Rule Logic:  
+
+- For each pump in B_RMR: `for pump_b in B_RMR.ASHRAE229.pumps:`
+
+  - Save loop served by pump and pump to a dictionary: `loop_pump_dictionary[pump_b.loop_or_piping].append(pump)`
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.ASHRAE229.chillers:`
+
+  - Get cooling loop that chiller servers: `chw_loop_b = chiller_b.cooling_loop`
+
+    - Add chiller rated capacity to CHW loop total capacity: `chw_loop_capacity_dict[chw_loop_b] += chiller_b.rated_capacity`
+
+- For each CHW loop that chiller serves: `for loop_b in chw_loop_capacity_dict.keys():`
+
```

Comment:
If fule 22-7 passes there would be only one CHW loop connected to all chillers.

Reply:
If Rule 22-7 does not pass, do we still need to check this rule?

Reply:
if 22-7 doesn't pass then 22-8 would not be applicable and should not be tested. 
@mkarpman - do you agree?